improver_tests/spotdata/spotdata/test_NeighbourSelection.py

```diff
@@ -517,7 +518,8 @@ def test_only_land(self):
         result, result_nodes = plugin.build_KDTree(self.region_land_mask)
         expected_length = np.nonzero(self.region_land_mask.data)[0].shape[0]
 
-        self.assertEqual(result_nodes.shape[0], expected_length)
+        self.assertEqual(result_nodes.shape[0],  # pylint: disable=E1136
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Id 'E1136' is used to disable 'unsubscriptable-object' message emission](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=4915701)